schema/cassandra/cadence/schema.cql

```diff
@@ -404,6 +409,7 @@ CREATE TABLE domains_by_name_v2 (
   config_version                bigint, -- indicating the version of domain config, excluding the failover / change of active cluster name
   failover_version              bigint, -- indicating the version of active domain only, used for domain failover
   failover_notification_version bigint, -- indicating the last change related to domain failover
+  failover_config               frozen<domain_failover_config>, -- indicating domain failover state
```

Comment:
On the naming things. I feel failover_state might be more accurate in this case?
Why not just get rid of this struct and unwrap failover_start_time and failover_timeout?

Reply:
Removed the struct and only use failover end time